example/centos/centos-9-aarch64-image-installer.yaml

```diff
@@ -0,0 +1,572 @@
+              - plymouth
+              - crypt
+              - dm
+              - dmsquash-live
+              - kernel-modules
+              - kernel-modules-extra
+              - kernel-network-modules
+              - livenet
+              - lvm
+              - mdraid
+              - qemu
+              - qemu-net
+              - resume
+              - rootfs-block
+              - terminfo
+              - udev-rules
+              - dracut-systemd
+              - pollcdrom
+              - usrmount
+              - base
+              - fs-lib
+              - img-lib
+              - shutdown
+              - uefi-lib
+              - nvdimm
+              - prefixdevname
+              - prefixdevname-tools
+              - anaconda
+              - rdma
+              - rngd
+              - multipath
+              - fcoe
+              - fcoe-uefi
+              - iscsi
+              - lunmask
+              - nfs
+            add_drivers:
+              - cuse
+              - ipmi_devintf
+              - ipmi_msghandler
+            install:
+              - /.buildstamp
+        - type: org.osbuild.selinux.config
+          options:
+            state: permissive
+    - name: rootfs-image
+      build: name:build
+      stages:
+        - type: org.osbuild.mkdir
+          options:
+            paths:
+              - path: /LiveOS
+        - type: org.osbuild.truncate
+          options:
+            filename: /LiveOS/rootfs.img
+            size: '5368709120'
+        - type: org.osbuild.mkfs.ext4
+          options:
+            uuid: 2fe99653-f7ff-44fd-bea8-fa70107524fb
+            label: Anaconda
+          devices:
+            device:
+              type: org.osbuild.loopback
+              options:
+                filename: LiveOS/rootfs.img
+        - type: org.osbuild.copy
+          inputs:
+            tree:
+              type: org.osbuild.tree
+              origin: org.osbuild.pipeline
+              references:
+                - name:anaconda-tree
+          options:
+            paths:
+              - from: input://tree/
+                to: mount://device/
+          devices:
+            device:
+              type: org.osbuild.loopback
+              options:
+                filename: LiveOS/rootfs.img
+          mounts:
+            - name: device
+              type: org.osbuild.ext4
+              source: device
+              target: /
+    - name: efiboot-tree
+      build: name:build
+      stages:
+        - type: org.osbuild.grub2.iso
+          options:
+            product:
+              name: CentOS Stream
+              version: 9-stream
+            kernel:
+              dir: /images/pxeboot
+              opts:
+                - inst.stage2=hd:LABEL=CentOS-Stream-9-BaseOS-aarch64
+                - inst.ks=hd:LABEL=CentOS-Stream-9-BaseOS-aarch64:/osbuild.ks
+            isolabel: CentOS-Stream-9-BaseOS-aarch64
+            architectures:
+              - AA64
```

Comment:
This is also interesting - so `grub2` has it's own set of architecture codes?

Reply:
Yes.